cmd/minikube/cmd/service.go

```diff
@@ -44,9 +48,15 @@ var serviceCmd = &cobra.Command{
 		}
 
 		service := args[0]
-
 		api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 		defer api.Close()
+
+		cluster.EnsureMinikubeRunningOrExit(api)
+		if err := commonutil.RetryAfter(20, func() error { return CheckService(namespace, service) }, 6*time.Second); err != nil {
+			fmt.Println("Could not find finalized endpoint being pointed to by %s: %s", service, err)
```

Comment:
Could you make sure all this logging goes to os.Stderr? The output of this command needs to be parseable.